Modules/_io/_iomodule.c

```diff
@@ -509,7 +509,7 @@ _io_text_encoding_impl(PyObject *module, PyObject *encoding, int stacklevel)
 /*[clinic end generated code: output=91b2cfea6934cc0c input=bf70231213e2a7b4]*/
 {
     if (encoding == NULL || encoding == Py_None) {
-        PyInterpreterState *interp = _PyInterpreterState_GET();
+        PyInterpreterState *interp = PyInterpreterState_Get();
```

Comment:
What the switch to `PyInterpreterState_Get()`?

Reply:
Modules cannot access a thread-local directly because they are dynamically loaded. It is a loader, rather a language, limitation, I think.

Reply:
That's good to know.  Thanks for explaining.  This might not be obvious to maintainers.  Would you mind adding a short comment next to the declaration of `_py_current_interpreter` noting the constraint?

Also, this doesn't affect builtin modules, right?  Aren't some of the modules updated here builtin modules?